qemu/tests/getfd.py

```diff
@@ -52,14 +52,14 @@ def has_fd(pid, filepath):
         path = os.path.join(fdfiles_dir, name)
         fd = os.open(path, os.O_RDWR | os.O_CREAT)
         response = vm.monitor.getfd(fd, name)
-        os.close(fd)
```

Comment:
@luckyh  thanks a lot for your explanation.

I have updated the code according to your comments,
could you help review it again ? thanks.